Services/AdvancedMetaData/classes/class.ilAdvancedMDRecord.php

```diff
@@ -174,8 +174,15 @@ public static function _getAssignableObjectTypes($a_include_text = false)
 		$lng = $DIC['lng'];
 		
 		$types = array();
+		$filter = array();
 		$amet_types = $objDefinition->getAdvancedMetaDataTypes();
 
+		include_once './Services/WebServices/ECS/classes/class.ilECSSetting.php';
+		if(!ilECSSetting::_ecsConfigured()){
+			include_once './Services/WebServices/ECS/classes/class.ilECSEventQueueReader.php';
+			$filter = array_merge($filter ,ilECSEventQueueReader::getAllEContentTypes());
```

Comment:
Please use: ilECSUtils:getPossibleRemoteTypes(false) and add the type rtst to it.